storage/src/main/java/tech/pegasys/artemis/storage/ChainStorageClient.java

```diff
@@ -261,10 +261,23 @@ public boolean isIncludedInBestState(final Bytes32 blockRoot) {
     return Optional.of(BeaconStateUtil.get_block_root_at_slot(bestState, slot));
   }
 
+  // TODO: These methods should not return zero if null. We should handle this better
```

Comment:
I think 0 is right in the cases `getFinalizedEpoch` was intended for.  You can check `isPregenesis` if you specifically want to know if the store is inited or not - but otherwise treating the finalised and justified epoch as 0 is reasonable.  Not so much for the zero bytes...

Reply:
done.